### DISCUSSION TITLE
Token duplication

### DISCUSSION THREAD

**Discussion by 13151719** [pocketbase/pocketbase#1577] [Q&A]
When a client refreshes the `pb.collection('users').authRefresh()` model and someone else logs in at the same time using oauth2 (in my case via Discord) pocketbase assigns a token to the refreshing client giving access to someone else's data.

**Comment by ganigeorgiev** [comment]
I don't understand what do you mean.

PocketBase doesn't have sessions. `pb.collection('users').authRefresh()` refreshes the existing token and user data in the local `pb.authStore`.

If you are integrating in a SSR environment, you need a different PocketBase SDK client instance for each server-side request (this is noted in https://github.com/pocketbase/js-sdk#ssr-integration).

Could you provide a code sample or minimal reproducible repo where you are experiencing the error?

**Reply by 13151719** [reply]
I created the app from scratch and it seems to be working at this point, I will examine my code more thoroughly and let you know, but it still seems strange to me because I create a new pocketbase instance every request.
